api/specs/webserver/openapi-user.yaml

```diff
@@ -101,7 +101,154 @@ paths:
         "204":
           description: token has been successfully deleted
 
+  /me/notifications:
```

Comment:
It would be great if you can create the corresponding ``api/specs/webserver/scripts/openapi_user.py`` for these specs. We are in the process of transforming all the OAS to python so they can more reliably generated and in addition the models can be reused in the code logic